BoxBrowseSDK/BoxBrowseSDK.xcodeproj/project.pbxproj

```diff
@@ -224,15 +224,15 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				C1E7651D66F260349D91B959 /* libPods.a in Frameworks */,
+				A3DAF34160188D517F8283B9 /* Pods.framework in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 		15146EB81AC20B78000BA380 /* Frameworks */ = {
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				1C6E5394A7F19AE4673FFCCC /* libPods.a in Frameworks */,
+				1C6E5394A7F19AE4673FFCCC /* (null) in Frameworks */,
```

Comment:
There have been other things in the past over the years that left (null) entries in the xcode xml. What we've done in the past to clean it up is edit manually (search for '(null)' and delete that whole line).
Can do that here, though I'm not sure if you're saying cocoapods is likely to re-introduce that every time...?